<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>429</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>429</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>429</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BatchCrypt</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QWidget" name="wOptions" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Директория входных файлов</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="leInDir"/>
           </item>
           <item>
            <widget class="QPushButton" name="pbInDirOpen">
             <property name="text">
              <string>Открыть</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Директория выходных файлов</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="leOutDir"/>
           </item>
           <item>
            <widget class="QPushButton" name="pbOutDirOpen">
             <property name="text">
              <string>Открыть</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Маска входных файлов</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leMask">
           <property name="text">
            <string>*</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Если выходной файл существует</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cobFileExistAction">
           <item>
            <property name="text">
             <string>Перезаписать</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Добавить счётчик к имени</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Ключ шифрования</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leKey">
           <property name="inputMask">
            <string>HH HH HH HH HH HH HH HH</string>
           </property>
           <property name="text">
            <string>00 00 00 00 00 00 00 00</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="cbDelete">
         <property name="text">
          <string>Удалять входные файлы после шифрования</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbTimer">
         <property name="text">
          <string>Работа по таймеру</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="wTimer" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>30</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lbTimer">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Периодичность опроса</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <widget class="QSpinBox" name="sbInterval">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>2147483647</number>
              </property>
              <property name="value">
               <number>5</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cobIntervalUnit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="currentIndex">
               <number>2</number>
              </property>
              <item>
               <property name="text">
                <string>ч</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>мин </string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>сек</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>мс</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbProcess">
        <property name="text">
         <string>Выполнить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
